gammapy/spectrum/tests/test_flux_point.py

```diff
@@ -79,6 +82,7 @@ def model(x):
 
 
 @requires_dependency('scipy')
+@pytest.mark.xfail(reason='I dont understand why we support this (joleroi)')
```

Comment:
@cdeil @JonathanDHarris (I don't know who of you worked on this)

Can you tell me what this test is good for? I looks like it test for funny looking array shapes, I don't know how to support this, maybe it's easier for me when I understand what it is supposed to do exactly.